src/legacy/core_plugins/kibana/public/management/sections/objects/lib/resolve_saved_objects.ts

```diff
@@ -18,9 +18,17 @@
  */
 
 import { i18n } from '@kbn/i18n';
+import { OverlayStart } from 'src/core/public';
+import {
+  SavedObject,
+  SavedObjectLoader,
+} from '../../../../../../../../plugins/saved_objects/public';
+import { IndexPatternsContract, IIndexPattern } from '../../../../../../../../plugins/data/public';
 
-async function getSavedObject(doc, services) {
-  const service = services.find(service => service.type === doc._type);
+type SavedObjectsRawDoc = Record<string, any>;
```

Comment:
Couldn't find a proper type for the raw doc we are manipulating in this file. Even the server side `SavedObjectsRawDoc` doesn't match. Had to fallback to a generic record...